docs/GOVERNANCE.md

```diff
@@ -0,0 +1,207 @@
+# Repository Governance
+
+This document defines the governance structure for the Kubeflow Pipelines Components Repository.
+
+## Table of Contents
+
+- [Two-Tier System](#two-tier-system)
+- [Ownership Models](#ownership-models)
+- [Tier Transitions](#tier-transitions)
+- [Removal Policies](#removal-policies)
+- [Deprecation Policy](#deprecation-policy)
+- [Repository Roles](#repository-roles)
+- [Decision Making](#decision-making)
+- [Conflict Resolution](#conflict-resolution)
+- [Policy Updates](#policy-updates)
+- [Related Documentation](#related-documentation)
+- [Background](#background)
+
+## Two-Tier System
+
+*Classification system distinguishing officially supported components from community contributions.*
+
+### Core Tier
+
+**Officially supported components** maintained by the Kubeflow community.
```

Comment:
The word "community" is becoming a big ambiguous, because you say Core Tier is supposted by Kubeflow **Community**, whereas Third Party components are **Community**-contributed components with lighter requirements. I think people will start getting confused. 

I propose we use the following explicit terminology: 

Core Tier: Kubeflow Core Maintainers. (define what this means) 
Third Party Tier:  Kubeflow Third Party Maintainers  (define what this means) 


You should also mention that the Root governance of the Repository - for example the APPROVERS in the owners file at the root repository lies with primarily KFP maintainers. This is not the same as Core maitnainers. We can address whether we want to expand this governance later.